File: src/IsPrime.java
import java.util.Scanner;

public class IsPrime {
    Scanner sc = new Scanner(System.in);
    int numero, contador = 0;
    public void Primo(){
        System.out.println("Digite o número");
        numero = sc.nextInt();
        for(int i = 2; i<numero; i++){
            if(numero%i == 0){
                contador++;
            }
        }
        if(contador>0){
            System.out.println("O número não é primo");
        }
        else{
            System.out.println("O número é primo");
        }  
    }
}

File: js/isprime.js
function isPrime(num) {
    // Um número menor que 2 não pode ser primo
    if (num < 2) {
      return false;
    }
  
    // Percorremos todos os números menores que num para verificar se é divisível
    for (let i = 2; i < num; i++) {
      // Se o número for divisível por algum outro número, não é primo
      if (num % i === 0) {
        return false;
      }
    }
  
    // Se nenhum número menor que num foi divisível, ele é primo
    return true;
  }